[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '40910']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-02T00:30:05.212Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -port 40910
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-02T00:30:05.212Z	INFO	first_node	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.425] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.430] New local node record                    seq=1,735,777,807,429 id=559d0556aa8bdf84 ip=127.0.0.1 udp=0 tcp=37283
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - ERROR[01-02|00:30:07.430] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.430] Started P2P networking                   self="enode://65ab7f172a566a3775fcc8c878b104afdad28b76d4b77f3b993cdc8f11490870a389f1d3e949bbebf9cc300e5028b91bed04dd7960b30113af9c8ab57a4f4742@127.0.0.1:37283?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.431] HTTP server started                      endpoint=127.0.0.1:40910 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.453] New local node record                    seq=1,735,777,807,435 id=c389441fe302949f ip=10.1.0.31 udp=0 tcp=33415
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.454] New local node record                    seq=1,735,777,807,436 id=c389441fe302949f ip=10.1.0.31 udp=55642 tcp=33415
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.456] New local node record                    seq=1,735,777,807,437 id=c389441fe302949f ip=10.1.0.31 udp=55642 tcp=33415
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-02T00:30:07.894Z	INFO	first_node	status-cli/util.go:68	account created, key UID: 0x2854b2655d13c7be75995369903a9170f063c8296b513404f48b08419e91f55f
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:07.895] New local node record                    seq=1,735,777,807,438 id=c389441fe302949f ip=10.1.0.31 udp=55642 tcp=33415
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-02T00:30:07.924Z	INFO	first_node	status-cli/util.go:93	messenger started, public key: 0x04baa8ce2cccd3ccd3f1eb8969334c64b04ed1172947fd256420e10f01e1efe85787c44c6e246ef0351aebe8e3c8ed513b2e5360f02f20183f50bd68ca29d9b3e6
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [01-02|00:30:12.450] New local node record                    seq=1,735,777,807,439 id=c389441fe302949f ip=172.183.175.192 udp=55642 tcp=37771
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-01-02T00:30:12.925Z	INFO	first_node	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131', '-kid', '0x3b37564e663adeaa98537e7feda63dd3abc98654d1e93ee22283bb5d75da1ead']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:13.227Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - -name 0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - -key-uid 0x3b37564e663adeaa98537e7feda63dd3abc98654d1e93ee22283bb5d75da1ead
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:13.227Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.281] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - ERROR[01-02|00:30:13.289] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.289] New local node record                    seq=1,727,788,918,300 id=9ac808199823a0eb ip=127.0.0.1 udp=0 tcp=40315
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.289] Started P2P networking                   self="enode://366246209c5b872e1aa920d3ab72272857edc4d85c78f0a6724bda35332b0b0084aded52eae6d53a1af57895fae210d1fb607aba4db9358fa56e61b990433b9c@127.0.0.1:40315?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.290] HTTP server started                      endpoint=127.0.0.1:6131 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.313] New local node record                    seq=1,735,777,813,293 id=9e45d20168afb083 ip=10.1.0.31 udp=0 tcp=45791
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.315] New local node record                    seq=1,735,777,813,294 id=9e45d20168afb083 ip=10.1.0.31 udp=36014 tcp=45791
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.318] New local node record                    seq=1,735,777,813,295 id=9e45d20168afb083 ip=10.1.0.31 udp=36014 tcp=45791
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:13.456Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/util.go:62	existing account, key UID: 0x3b37564e663adeaa98537e7feda63dd3abc98654d1e93ee22283bb5d75da1ead
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:13.457] New local node record                    seq=1,735,777,813,296 id=9e45d20168afb083 ip=10.1.0.31 udp=36014 tcp=45791
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:13.476Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/util.go:93	messenger started, public key: 0x04f9be3ecf73726466655b09c261c7cc3b5bd99ca4b5b10fee6768970ee93e7f004ac362e30b66944c16a1137de5d6955d0ec3a80e72970eb0c3f032985ab08cc0
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136', '-kid', '0xb4e057eeae4db5ea877a4280c0b283b66aa7d166cafa34a296236add87d62ae8']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - 2025-01-02T00:30:15.227Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - -name 0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - -key-uid 0xb4e057eeae4db5ea877a4280c0b283b66aa7d166cafa34a296236add87d62ae8
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - 2025-01-02T00:30:15.227Z	INFO	0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.281] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - ERROR[01-02|00:30:15.288] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.288] New local node record                    seq=1,727,788,963,926 id=c47d3ec3d9408d8d ip=127.0.0.1 udp=0 tcp=37331
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.288] Started P2P networking                   self="enode://e2f58bef4853d045ae3b7029c10da1acd43702b1b725b9df294d9df01a42beb9cdb6902f860a00971a7c9fd54b8f0cc6b6520c327317487d9930fe63b272609f@127.0.0.1:37331?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.289] HTTP server started                      endpoint=127.0.0.1:6136 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.317] New local node record                    seq=1,735,777,815,292 id=9d86d7e3b11c330a ip=10.1.0.31 udp=42526 tcp=37831
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.319] New local node record                    seq=1,735,777,815,293 id=9d86d7e3b11c330a ip=10.1.0.31 udp=42526 tcp=37831
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - 2025-01-02T00:30:15.455Z	INFO	0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136	status-cli/util.go:62	existing account, key UID: 0xb4e057eeae4db5ea877a4280c0b283b66aa7d166cafa34a296236add87d62ae8
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:15.456] New local node record                    seq=1,735,777,815,294 id=9d86d7e3b11c330a ip=10.1.0.31 udp=42526 tcp=37831
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - 2025-01-02T00:30:15.475Z	INFO	0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136	status-cli/util.go:93	messenger started, public key: 0x045bb4b679cf59e10ad9668a6c96413b4b966726b3e70e196db21a1013a982357fc9f6b5dc4cda8f9f1763cb0097b06c12906ead9c2149e4e7b34942cac7ead355
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.480] Served personal_ecRecover                reqid=1 duration="47.389µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.481] Served personal_ecRecover                reqid=2 duration="21.79µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.483] Served personal_ecRecover                reqid=3 duration="19.537µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.484] Served personal_ecRecover                reqid=4 duration="19.316µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.486] Served personal_ecRecover                reqid=5 duration="20.568µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.487] Served personal_ecRecover                reqid=6 duration="26.841µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.488] Served personal_ecRecover                reqid=7 duration="23.624µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.489] Served personal_ecRecover                reqid=8 duration="27.061µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.491] Served personal_ecRecover                reqid=9 duration="19.206µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.492] Served personal_ecRecover                reqid=10 duration="25.318µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.493] Served personal_ecRecover                reqid=11 duration="25.017µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.494] Served personal_ecRecover                reqid=12 duration="25.728µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.496] Served personal_ecRecover                reqid=13 duration="24.936µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.497] Served personal_ecRecover                reqid=14 duration="27.301µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.498] Served personal_ecRecover                reqid=15 duration="27.431µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.499] Served personal_ecRecover                reqid=16 duration="23.995µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.501] Served personal_ecRecover                reqid=17 duration="27.001µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.502] Served personal_ecRecover                reqid=18 duration="21.43µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.504] Served personal_ecRecover                reqid=19 duration="19.216µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.505] Served personal_ecRecover                reqid=20 duration="24.856µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.506] Served personal_ecRecover                reqid=21 duration="18.704µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.507] Served personal_ecRecover                reqid=22 duration="18.424µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.509] Served personal_ecRecover                reqid=23 duration="18.414µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.510] Served personal_ecRecover                reqid=24 duration="24.537µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.511] Served personal_ecRecover                reqid=25 duration="23.243µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.513] Served personal_ecRecover                reqid=26 duration="18.855µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.514] Served personal_ecRecover                reqid=27 duration="18.224µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.515] Served personal_ecRecover                reqid=28 duration="18.465µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.516] Served personal_ecRecover                reqid=29 duration="18.394µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.518] Served personal_ecRecover                reqid=30 duration="22.041µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.519] Served personal_ecRecover                reqid=31 duration="18.905µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.520] Served personal_ecRecover                reqid=32 duration="19.426µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.522] Served personal_ecRecover                reqid=33 duration="23.043µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.523] Served personal_ecRecover                reqid=34 duration="19.085µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.524] Served personal_ecRecover                reqid=35 duration="26.741µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.526] Served personal_ecRecover                reqid=36 duration="23.785µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.528] Served personal_ecRecover                reqid=37 duration="23.183µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.529] Served personal_ecRecover                reqid=38 duration="23.364µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.530] Served personal_ecRecover                reqid=39 duration="25.577µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.532] Served personal_ecRecover                reqid=40 duration="26.008µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.533] Served personal_ecRecover                reqid=41 duration="27.08µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.535] Served personal_ecRecover                reqid=42 duration="22.362µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.536] Served personal_ecRecover                reqid=43 duration="23.394µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.537] Served personal_ecRecover                reqid=44 duration="25.558µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.539] Served personal_ecRecover                reqid=45 duration="25.377µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.540] Served personal_ecRecover                reqid=46 duration="25.137µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.541] Served personal_ecRecover                reqid=47 duration="25.538µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.542] Served personal_ecRecover                reqid=48 duration="25.838µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.544] Served personal_ecRecover                reqid=49 duration="25.247µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.545] Served personal_ecRecover                reqid=50 duration="25.017µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.546] Served personal_ecRecover                reqid=51 duration="25.026µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.548] Served personal_ecRecover                reqid=52 duration="27.432µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.550] Served personal_ecRecover                reqid=53 duration="26.8µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.551] Served personal_ecRecover                reqid=54 duration="26.259µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.553] Served personal_ecRecover                reqid=55 duration="26.961µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.554] Served personal_ecRecover                reqid=56 duration="20.158µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.555] Served personal_ecRecover                reqid=57 duration="25.758µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.556] Served personal_ecRecover                reqid=58 duration="24.516µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.558] Served personal_ecRecover                reqid=59 duration="25.909µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.559] Served personal_ecRecover                reqid=60 duration="28.303µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.560] Served personal_ecRecover                reqid=61 duration="24.837µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.561] Served personal_ecRecover                reqid=62 duration="18.434µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.563] Served personal_ecRecover                reqid=63 duration="26.229µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.564] Served personal_ecRecover                reqid=64 duration="21.029µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.565] Served personal_ecRecover                reqid=65 duration="22.131µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.567] Served personal_ecRecover                reqid=66 duration="19.075µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.568] Served personal_ecRecover                reqid=67 duration="21.44µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.569] Served personal_ecRecover                reqid=68 duration="38.803µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.570] Served personal_ecRecover                reqid=69 duration="21.69µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.572] Served personal_ecRecover                reqid=70 duration="23.544µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.573] Served personal_ecRecover                reqid=71 duration="24.957µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.575] Served personal_ecRecover                reqid=72 duration="52.228µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.577] Served personal_ecRecover                reqid=73 duration="26.831µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.579] Served personal_ecRecover                reqid=74 duration="22.252µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.580] Served personal_ecRecover                reqid=75 duration="26.089µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.581] Served personal_ecRecover                reqid=76 duration="24.396µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:16.582] Served personal_ecRecover                reqid=77 duration="26.97µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138', '-kid', '0x381cd65f19f50af924e50033c56dd17a3bdbdd8622ac7de38657a11618b8e881']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - 2025-01-02T00:30:17.232Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - -name 0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - -key-uid 0x381cd65f19f50af924e50033c56dd17a3bdbdd8622ac7de38657a11618b8e881
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - 2025-01-02T00:30:17.233Z	INFO	0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.287] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - ERROR[01-02|00:30:17.294] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.295] New local node record                    seq=1,727,788,982,191 id=33c271785e99a5eb ip=127.0.0.1 udp=0 tcp=44141
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.295] Started P2P networking                   self="enode://f9196f44644a0758153110a9b3a7439e60dd1f7aae7d2d6cdf98d921b40c07e8c8814d73f89926b31aecf7a8b768f7c1320ddf32c53d13d259fae14305b21f40@127.0.0.1:44141?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.295] HTTP server started                      endpoint=127.0.0.1:6138 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.328] New local node record                    seq=1,735,777,817,299 id=c9958765038cf26c ip=10.1.0.31 udp=34568 tcp=40911
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.329] New local node record                    seq=1,735,777,817,300 id=c9958765038cf26c ip=10.1.0.31 udp=34568 tcp=40911
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - 2025-01-02T00:30:17.469Z	INFO	0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138	status-cli/util.go:62	existing account, key UID: 0x381cd65f19f50af924e50033c56dd17a3bdbdd8622ac7de38657a11618b8e881
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:17.470] New local node record                    seq=1,735,777,817,301 id=c9958765038cf26c ip=10.1.0.31 udp=34568 tcp=40911
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.481] Served personal_ecRecover                reqid=78 duration="62.157µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.485] Served personal_ecRecover                reqid=79 duration="48.441µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.487] Served personal_ecRecover                reqid=80 duration="28.523µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.489] Served personal_ecRecover                reqid=81 duration="33.824µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.493] Served personal_ecRecover                reqid=82 duration="30.707µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.495] Served personal_ecRecover                reqid=83 duration="19.988µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - 2025-01-02T00:30:17.495Z	INFO	0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138	status-cli/util.go:93	messenger started, public key: 0x04ee16839ad4a02acb8245cfd42d13aca6302bdceab767e73ef3a9bbef9e9b25b3dfa5ffbfe6231d263cee6d2f7fba0f119abcc6ee3753788c002368c0c3706a30
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.498] Served personal_ecRecover                reqid=84 duration="22.682µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.500] Served personal_ecRecover                reqid=85 duration="25.858µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.501] Served personal_ecRecover                reqid=86 duration="22.532µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.502] Served personal_ecRecover                reqid=87 duration="19.597µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.505] Served personal_ecRecover                reqid=88 duration="26.56µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.508] Served personal_ecRecover                reqid=89 duration="20.238µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.509] Served personal_ecRecover                reqid=90 duration="19.767µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.512] Served personal_ecRecover                reqid=91 duration="20.498µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.516] Served personal_ecRecover                reqid=92 duration="22.802µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.517] Served personal_ecRecover                reqid=93 duration="24.756µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.518] Served personal_ecRecover                reqid=94 duration="19.957µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.521] Served personal_ecRecover                reqid=95 duration="21.59µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.523] Served personal_ecRecover                reqid=96 duration="21.22µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.524] Served personal_ecRecover                reqid=97 duration="20.768µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.525] Served personal_ecRecover                reqid=98 duration="24.405µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.527] Served personal_ecRecover                reqid=99 duration="48.801µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - WARN [01-02|00:30:17.530] Served personal_ecRecover                reqid=100 duration="27.211µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:18.308] New local node record                    seq=1,735,777,813,297 id=9e45d20168afb083 ip=172.183.175.192 udp=36014 tcp=37760
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:18.476Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139', '-kid', '0xe38a5af51deca8771d81bc5bdd1d2fc7d970d68fe161b9012c5dc663f3f2d4b0']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - 2025-01-02T00:30:19.234Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - -name 0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - -key-uid 0xe38a5af51deca8771d81bc5bdd1d2fc7d970d68fe161b9012c5dc663f3f2d4b0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - 2025-01-02T00:30:19.234Z	INFO	0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.304] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.311] New local node record                    seq=1,727,788,991,302 id=9a0e42401d062218 ip=127.0.0.1 udp=0 tcp=38759
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.311] Started P2P networking                   self="enode://634b69d13dc4d2de9aef02e6b7137a2e2f1e3287d55b6e8c554ca9d92f2d4ce0200e13d1867a2a11ed26172701df1c4a1cf7376706cd73e0bae188a2b5d2e414@127.0.0.1:38759?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - ERROR[01-02|00:30:19.312] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.313] HTTP server started                      endpoint=127.0.0.1:6139 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.354] New local node record                    seq=1,735,777,819,323 id=6862f44e235c7a3a ip=10.1.0.31 udp=0 tcp=43915
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.356] New local node record                    seq=1,735,777,819,324 id=6862f44e235c7a3a ip=10.1.0.31 udp=35697 tcp=43915
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.358] New local node record                    seq=1,735,777,819,325 id=6862f44e235c7a3a ip=10.1.0.31 udp=35697 tcp=43915
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - 2025-01-02T00:30:19.538Z	INFO	0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139	status-cli/util.go:62	existing account, key UID: 0xe38a5af51deca8771d81bc5bdd1d2fc7d970d68fe161b9012c5dc663f3f2d4b0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:19.539] New local node record                    seq=1,735,777,819,326 id=6862f44e235c7a3a ip=10.1.0.31 udp=35697 tcp=43915
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - 2025-01-02T00:30:19.566Z	INFO	0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139	status-cli/util.go:93	messenger started, public key: 0x04c02626669e62a78419296d13a637bace2fa91bbf26bab96a4bf71cf68265c2cb392f3af427156676ebc9531df3aff38a5f941d7c312b86e37ccd6e7de8506dc5
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - INFO [01-02|00:30:20.310] New local node record                    seq=1,735,777,815,295 id=9d86d7e3b11c330a ip=172.183.175.192 udp=42526 tcp=37761
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - 2025-01-02T00:30:20.476Z	INFO	0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.483] Served personal_ecRecover                reqid=1 duration="54.071µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.485] Served personal_ecRecover                reqid=2 duration="23.183µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.487] Served personal_ecRecover                reqid=3 duration="22.923µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.489] Served personal_ecRecover                reqid=4 duration="21.029µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.491] Served personal_ecRecover                reqid=5 duration="20.979µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.492] Served personal_ecRecover                reqid=6 duration="20.87µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.494] Served personal_ecRecover                reqid=7 duration="21.761µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.500] Served personal_ecRecover                reqid=8 duration="26.6µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.501] Served personal_ecRecover                reqid=9 duration="23.293µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.503] Served personal_ecRecover                reqid=10 duration="21.119µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.505] Served personal_ecRecover                reqid=11 duration="69.38µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.508] Served personal_ecRecover                reqid=12 duration="21.34µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.510] Served personal_ecRecover                reqid=13 duration="30.537µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.511] Served personal_ecRecover                reqid=14 duration="25.387µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.516] Served personal_ecRecover                reqid=15 duration="54.542µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.518] Served personal_ecRecover                reqid=16 duration="19.646µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.519] Served personal_ecRecover                reqid=17 duration="20.559µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.522] Served personal_ecRecover                reqid=18 duration="94.937µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.524] Served personal_ecRecover                reqid=19 duration="21.871µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.526] Served personal_ecRecover                reqid=20 duration="22.162µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.528] Served personal_ecRecover                reqid=21 duration="22.402µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.529] Served personal_ecRecover                reqid=22 duration="20.318µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.535] Served personal_ecRecover                reqid=23 duration="43.421µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.537] Served personal_ecRecover                reqid=24 duration="20.298µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.539] Served personal_ecRecover                reqid=25 duration="25.657µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.542] Served personal_ecRecover                reqid=26 duration="27.131µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.544] Served personal_ecRecover                reqid=27 duration="49.412µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.549] Served personal_ecRecover                reqid=28 duration="22.382µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02AFC3FE72D25D01723D97275041E5C000248CA36851C4B7947623561DC957F618_6136 - WARN [01-02|00:30:20.551] Served personal_ecRecover                reqid=29 duration="22.352µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130', '-kid', '0xdeae06f118e333ee383ea227f1310aa78c41068e95145583e90c1637ea61aa99']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - 2025-01-02T00:30:21.235Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - -name 0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - -key-uid 0xdeae06f118e333ee383ea227f1310aa78c41068e95145583e90c1637ea61aa99
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - 2025-01-02T00:30:21.235Z	INFO	0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.314] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.321] New local node record                    seq=1,727,788,904,679 id=a82bc60840bbedd8 ip=127.0.0.1 udp=0 tcp=37889
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.321] Started P2P networking                   self="enode://10316046411954dfad1622da44b4780d9d58585a0db23811f71d520113812aedc03c0242250b73418fb01854a82cb3b7a93df0bb3301979f7cb23f2c422521de@127.0.0.1:37889?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - ERROR[01-02|00:30:21.322] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.322] HTTP server started                      endpoint=127.0.0.1:6130 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.358] New local node record                    seq=1,735,777,821,327 id=4bf1427bfc7f63cb ip=10.1.0.31 udp=54019 tcp=35555
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.360] New local node record                    seq=1,735,777,821,328 id=4bf1427bfc7f63cb ip=10.1.0.31 udp=54019 tcp=35555
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - 2025-01-02T00:30:21.546Z	INFO	0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130	status-cli/util.go:62	existing account, key UID: 0xdeae06f118e333ee383ea227f1310aa78c41068e95145583e90c1637ea61aa99
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:21.549] New local node record                    seq=1,735,777,821,329 id=4bf1427bfc7f63cb ip=10.1.0.31 udp=54019 tcp=35555
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - 2025-01-02T00:30:21.580Z	INFO	0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130	status-cli/util.go:93	messenger started, public key: 0x04cc73b3e56c2ad33f184831f2826f69136e8bf1cf9b0e4aaa34cd742b17a7b95f332ca5d038b4eb46effd056e06e3335cb5f10eed2274ede353f56bab8fe558c1
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 84753b95-8aaf-463a-a247-0da01e735f6e_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_22 (ID=0x5ce2777e51bb83d05b375c10d54779639f216374c226f516e81c8c79c685bf37)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: f5d7a8ac-946d-454a-8dea-71ec773c04c5_message_from_first_node_7 (ID=0xecb7946633630bc0bf755d0cf4227540d78e61a428c9486672d7ab12dd13d39f)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 9c142767-1ddb-4f45-822f-a60c0741fec8_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_2 (ID=0x0462ecb5638bb8051d7bed68b8dad9ab337ecf4db9df2472d1e8e5bb2aaca8c0)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: c975d20b-8eeb-4505-9c6f-56075a6d41e4_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_0 (ID=0xab4c415a18cb84e460e5d06724cc50a5f246b55e63345f02d4bd080fc4af1d37)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 6e1e2b0d-568d-431a-a2ba-f45aad02b1d9_message_from_first_node_23 (ID=0x159b26dcdabbfb2e9d9e5619d4b90f4a4106b74720bb708fc3bce9ecbfe42e69)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: dcdaf710-16da-408d-98a2-3eb683072bd1_message_from_first_node_19 (ID=0x89663a2f30327255b296ebae91e284707539617b29e950f9f592bba32c0c314f)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: d59b7f33-6d86-4bf3-8b17-b0fdb35a8040_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_8 (ID=0x950d01f19b0bec41c9c9c72aa171c7926eb5bebc7e084543188558dd8ae7d18a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 125a5aa1-05eb-4902-b43e-7ea271720a25_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_18 (ID=0x11c26240d2733d1815a0d1a8b8b1d00da7818de613a2ca2018c773c93ab006e7)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 8241ca93-ef58-4324-8e24-e2d8c5a743df_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_10 (ID=0x8ef84f89241e9ebbcffc4d3bff49d8dbf6ecdc86b524a6a5566d9a83178289b4)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: d8d3c421-4ab3-45cd-bd95-46b8d9a839c7_message_from_first_node_9 (ID=0xcadf69f733081e6839e8b84853616445d66e48b309201e7e7d8fcc42de48be7a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 4ff7b02b-4ca3-4990-a892-70e357a0e06e_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_12 (ID=0xa065340421e28cbbc5b1c5cfa399f02d87aef92b245371b1fb0b7c915ef46471)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 94b009a7-452c-478b-861d-ea3d93b1331c_message_from_first_node_11 (ID=0xc93135e02f6256c6f97acd5b336556830bfe53072b2ea10325c5a3fa9f644972)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 733b4629-166d-4add-8b57-7ff15f1bcc95_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_4 (ID=0xb697e4ad05bc214d91bcefa500e59a26b15aeb0e5d077e9ace98abaa35f7d203)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 7f2d0fef-548e-463a-b1c2-8620b2d23975_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_20 (ID=0xf2dfacb4c116a4d10ad5aea3dcd206fc2d45b06479d05c7bc320330945ae8538)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 0158e9e7-7e58-483d-b511-2109b636c6c6_message_from_first_node_17 (ID=0x8c2ffb0ee33afe230ba4a26965b3b5fd7d095ef85abace44a0ee7e708dbdd479)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 55eb03ad-b7ea-48bd-8e61-404e50ff645c_message_from_first_node_13 (ID=0x90615bd02cd55a5dfab3aec77998af1c8239cbc364a1a076623d3bf9d8389db3)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: abfe2f69-bc63-4836-ac44-a354765d7e61_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_6 (ID=0xdebf2d48f97a3f997fe46dece2f3380c6cbd523f52c0e6353f9f5f4be855fbe8)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 24d2e8f4-d323-407a-9e72-f244513e0405_message_from_first_node_3 (ID=0xf0fb6c25f84476b593f7de5dec273fe6608ba26c6e55f1adf45421f8ecfcb00c)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 209dd65f-04d4-476e-a22e-bcb6e15c176e_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_24 (ID=0x6ae8d769cf8ec0202cf9b8e02421dc2526b42428ebedc3cd5a7a4a8e105bd952)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 66e7da30-978b-4b8a-bbd9-f6e068495b11_message_from_first_node_5 (ID=0xa2cca6a01d1718ac8059445af94dd4e7019d2c3766470b08fb15e41dfc56708f)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: ebd2d9de-dae9-49d6-b375-ea1e848c018b_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_14 (ID=0x652f3cddfa90aa9f272f301e3d5b1f1240d2a7e9fecb3341daf66fb8c970e3b1)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: b867a1ec-408e-443d-b093-6adc2fc16dfc_message_from_first_node_15 (ID=0x19d78c9dbc166260bd0ade3d323788ac82d5236b48ef859507a02ad55d2f80c5)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: f6f8ea7c-75e1-4053-824d-0dfef0f0d1af_message_from_first_node_21 (ID=0x21122617e1325332f4a3b613c4a960601b394d6e855309b39630bbdf3814af64)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: 3978331d-526b-47d0-9279-b23e0859dd99_message_from_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131_16 (ID=0x6c641974375e7cff5d0ab711a3bf7e9d300d79b5b44d5168f8c772cc4c1d7f13)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - 2025-01-02T00:30:21.939Z	INFO	0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131	status-cli/serve.go:70	message received: ffdee278-ca69-4de6-8b8e-fa0adaaf565c_message_from_first_node_1 (ID=0x2688bbb3b710ea3225ceb9f00f7d06c78ea26538f2ec2a7f1b02ecbf23091407)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:22.318] New local node record                    seq=1,735,777,817,302 id=c9958765038cf26c ip=172.183.175.192 udp=34568 tcp=37762
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - 2025-01-02T00:30:22.496Z	INFO	0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.522] Served personal_ecRecover                reqid=1 duration="72.275µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.524] Served personal_ecRecover                reqid=2 duration="23.874µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.525] Served personal_ecRecover                reqid=3 duration="41.738µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.527] Served personal_ecRecover                reqid=4 duration="25.067µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.528] Served personal_ecRecover                reqid=5 duration="40.556µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.531] Served personal_ecRecover                reqid=6 duration="21.35µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.532] Served personal_ecRecover                reqid=7 duration="22.472µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.534] Served personal_ecRecover                reqid=8 duration="20.969µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.535] Served personal_ecRecover                reqid=9 duration="17.132µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.537] Served personal_ecRecover                reqid=10 duration="17.663µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.539] Served personal_ecRecover                reqid=11 duration="18.966µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.540] Served personal_ecRecover                reqid=12 duration="22.622µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.541] Served personal_ecRecover                reqid=13 duration="17.223µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.543] Served personal_ecRecover                reqid=14 duration="18.034µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.544] Served personal_ecRecover                reqid=15 duration="18.334µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.546] Served personal_ecRecover                reqid=16 duration="19.006µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.548] Served personal_ecRecover                reqid=17 duration="22.431µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.549] Served personal_ecRecover                reqid=18 duration="22.743µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.551] Served personal_ecRecover                reqid=19 duration="21.791µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.555] Served personal_ecRecover                reqid=20 duration="22.723µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.556] Served personal_ecRecover                reqid=21 duration="22.743µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.558] Served personal_ecRecover                reqid=22 duration="17.363µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.562] Served personal_ecRecover                reqid=23 duration="17.583µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.567] Served personal_ecRecover                reqid=24 duration="18.545µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.568] Served personal_ecRecover                reqid=25 duration="16.591µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.570] Served personal_ecRecover                reqid=26 duration="16.952µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.571] Served personal_ecRecover                reqid=27 duration="16.731µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - WARN [01-02|00:30:22.572] Served personal_ecRecover                reqid=28 duration="18.484µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134', '-kid', '0xc307f0b364bd3a14c06e19dd6c8a79ade6bbfee2cc7f1a70923e43fbd1b4514a']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - 2025-01-02T00:30:23.234Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - -name 0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - -key-uid 0xc307f0b364bd3a14c06e19dd6c8a79ade6bbfee2cc7f1a70923e43fbd1b4514a
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - 2025-01-02T00:30:23.235Z	INFO	0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.295] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.304] New local node record                    seq=1,727,788,945,676 id=69438348dfbb31cc ip=127.0.0.1 udp=0 tcp=40943
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.304] Started P2P networking                   self="enode://5dbd4502d3777eaed0712bb0eb666ec37bc47b447e980e2fc051b89c9310b974d01140db788915cf01aabe910b671fabf8a73376ea89f437bed9ba3779218076@127.0.0.1:40943?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - ERROR[01-02|00:30:23.304] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.305] HTTP server started                      endpoint=127.0.0.1:6134 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.336] New local node record                    seq=1,735,777,823,309 id=ed3c4211fd88bc69 ip=10.1.0.31 udp=0 tcp=33285
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.339] New local node record                    seq=1,735,777,823,310 id=ed3c4211fd88bc69 ip=10.1.0.31 udp=48432 tcp=33285
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.341] New local node record                    seq=1,735,777,823,311 id=ed3c4211fd88bc69 ip=10.1.0.31 udp=48432 tcp=33285
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - 2025-01-02T00:30:23.490Z	INFO	0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134	status-cli/util.go:62	existing account, key UID: 0xc307f0b364bd3a14c06e19dd6c8a79ade6bbfee2cc7f1a70923e43fbd1b4514a
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:23.491] New local node record                    seq=1,735,777,823,312 id=ed3c4211fd88bc69 ip=10.1.0.31 udp=48432 tcp=33285
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - 2025-01-02T00:30:23.512Z	INFO	0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134	status-cli/util.go:93	messenger started, public key: 0x04457f448444e5030895c638f21cf1bdb9d69c25ce400d6cd133db29283eada5704a57a944ddd60698c5a466b8baed54d85226dae39b043f5448f0095bbbd7e397
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:24.351] New local node record                    seq=1,735,777,819,327 id=6862f44e235c7a3a ip=172.183.175.192 udp=35697 tcp=37764
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - 2025-01-02T00:30:24.566Z	INFO	0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132', '-kid', '0xe825e47f4e41919c36c4c43fe06f61d4f694721ce1168502cc077e2b58376ab6']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - 2025-01-02T00:30:25.235Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - -name 0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - -key-uid 0xe825e47f4e41919c36c4c43fe06f61d4f694721ce1168502cc077e2b58376ab6
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - 2025-01-02T00:30:25.235Z	INFO	0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.295] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - ERROR[01-02|00:30:25.304] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.304] New local node record                    seq=1,727,788,927,434 id=e9afe6ee53da8bfc ip=127.0.0.1 udp=0 tcp=35327
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.304] Started P2P networking                   self="enode://2081f0e4cf813441cd7e44b16e47e3c12de8a636f1e5662d8942216b029e8b85a31a7b4fcf19494e05d585f8599a5e4279137b103d76c0f9ddaca72f4149eecc@127.0.0.1:35327?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.305] HTTP server started                      endpoint=127.0.0.1:6132 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.340] New local node record                    seq=1,735,777,825,316 id=9f1d9b14b645ebbf ip=10.1.0.31 udp=37695 tcp=39529
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.342] New local node record                    seq=1,735,777,825,317 id=9f1d9b14b645ebbf ip=10.1.0.31 udp=37695 tcp=39529
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - 2025-01-02T00:30:25.504Z	INFO	0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132	status-cli/util.go:62	existing account, key UID: 0xe825e47f4e41919c36c4c43fe06f61d4f694721ce1168502cc077e2b58376ab6
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:25.510] New local node record                    seq=1,735,777,825,318 id=9f1d9b14b645ebbf ip=10.1.0.31 udp=37695 tcp=39529
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - 2025-01-02T00:30:25.534Z	INFO	0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132	status-cli/util.go:93	messenger started, public key: 0x04a2b208fdeac8b95bc0950d8bd91ab5abfbb3217f7882fdcf04fee8654766104a500f9f0909fd1be73acc378775cffd41976f449f2ffb4e01cbde30f353aa8b35
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.584] Served personal_ecRecover                reqid=1 duration="65.923µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.585] Served personal_ecRecover                reqid=2 duration="25.527µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.587] Served personal_ecRecover                reqid=3 duration="27.792µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.589] Served personal_ecRecover                reqid=4 duration="29.635µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.590] Served personal_ecRecover                reqid=5 duration="27.912µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.592] Served personal_ecRecover                reqid=6 duration="36.218µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.593] Served personal_ecRecover                reqid=7 duration="24.295µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.595] Served personal_ecRecover                reqid=8 duration="24.456µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.596] Served personal_ecRecover                reqid=9 duration="24.887µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.598] Served personal_ecRecover                reqid=10 duration="20.368µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.599] Served personal_ecRecover                reqid=11 duration="26.068µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.600] Served personal_ecRecover                reqid=12 duration="25.427µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.602] Served personal_ecRecover                reqid=13 duration="28.303µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.604] Served personal_ecRecover                reqid=14 duration="19.416µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.605] Served personal_ecRecover                reqid=15 duration="43.02µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.607] Served personal_ecRecover                reqid=16 duration="27.441µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.608] Served personal_ecRecover                reqid=17 duration="42.67µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.610] Served personal_ecRecover                reqid=18 duration="24.195µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.612] Served personal_ecRecover                reqid=19 duration="24.055µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.613] Served personal_ecRecover                reqid=20 duration="21.249µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:25.614] Served personal_ecRecover                reqid=21 duration="21.57µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - INFO [01-02|00:30:26.354] New local node record                    seq=1,735,777,821,330 id=4bf1427bfc7f63cb ip=172.183.175.192 udp=54019 tcp=37769
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.574] Served personal_ecRecover                reqid=22 duration="39.684µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.576] Served personal_ecRecover                reqid=23 duration="28.924µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0226CBC810A4C88C49DA9920B74287463556974170EB628CC791639263B7C9E324_6130 - 2025-01-02T00:30:26.580Z	INFO	0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.581] Served personal_ecRecover                reqid=24 duration="32.15µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.583] Served personal_ecRecover                reqid=25 duration="24.796µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.590] Served personal_ecRecover                reqid=26 duration="23.203µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.592] Served personal_ecRecover                reqid=27 duration="28.644µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - WARN [01-02|00:30:26.593] Served personal_ecRecover                reqid=28 duration="30.637µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137', '-kid', '0xf573186115e5ee9069d7ecec95694c80d2605775d307b059410b275445ffde9c']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - 2025-01-02T00:30:27.239Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - -name 0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - -key-uid 0xf573186115e5ee9069d7ecec95694c80d2605775d307b059410b275445ffde9c
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - 2025-01-02T00:30:27.239Z	INFO	0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.299] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.309] New local node record                    seq=1,727,788,973,055 id=55864dd925fc2809 ip=127.0.0.1 udp=0 tcp=39131
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.309] Started P2P networking                   self="enode://778f4f3858b9245ea8b1488fbd0515ec91204e02d07dc3e29a21274356f043ff6eef77fcf80db4e7d17aa94fd2ae4c39d139a97da80a5ad9f1c756ce41e1eae4@127.0.0.1:39131?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - ERROR[01-02|00:30:27.309] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.310] HTTP server started                      endpoint=127.0.0.1:6137 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.347] New local node record                    seq=1,735,777,827,321 id=ff13e6b5eba89c53 ip=10.1.0.31 udp=0 tcp=33181
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.348] New local node record                    seq=1,735,777,827,322 id=ff13e6b5eba89c53 ip=10.1.0.31 udp=54136 tcp=33181
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.350] New local node record                    seq=1,735,777,827,323 id=ff13e6b5eba89c53 ip=10.1.0.31 udp=54136 tcp=33181
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - 2025-01-02T00:30:27.503Z	INFO	0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137	status-cli/util.go:62	existing account, key UID: 0xf573186115e5ee9069d7ecec95694c80d2605775d307b059410b275445ffde9c
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:27.504] New local node record                    seq=1,735,777,827,324 id=ff13e6b5eba89c53 ip=10.1.0.31 udp=54136 tcp=33181
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - 2025-01-02T00:30:27.534Z	INFO	0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137	status-cli/util.go:93	messenger started, public key: 0x04ec4307c42514dd88c5beb6bda6a40a5e9e2a1bdff74308ecad175eb3627cb4ca69f69b2a3ae8e86729f399b289d923f82fe5b3c05b997b883c54ba3d9f4d42be
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03600E1152158616F5E0B6B8213BD9A2A0B52410F2AEF58398F02C1B681FBAD117_6131 - INFO [01-02|00:30:28.308] New local node record                    seq=1,735,777,813,298 id=9e45d20168afb083 ip=172.183.175.192 udp=36014 tcp=37760
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - INFO [01-02|00:30:28.331] New local node record                    seq=1,735,777,823,313 id=ed3c4211fd88bc69 ip=172.183.175.192 udp=48432 tcp=37770
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X022C9BB111279B1264EB3C86D11BAF886354BCFED8E7B46DB96871686F9F91D96E_6134 - 2025-01-02T00:30:28.512Z	INFO	0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133', '-kid', '0x94a79a7074a5cbfc27a6ee4fbe303fb7340f49f6a4bd14ab7757940237918d9e']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - 2025-01-02T00:30:29.236Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - -name 0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - -key-uid 0x94a79a7074a5cbfc27a6ee4fbe303fb7340f49f6a4bd14ab7757940237918d9e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - 2025-01-02T00:30:29.236Z	INFO	0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.297] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.306] New local node record                    seq=1,727,788,936,547 id=2725eebdccf07536 ip=127.0.0.1 udp=0 tcp=37233
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.306] Started P2P networking                   self="enode://b185c8abad52e07854f58e8ae8b6f81b1d79f68c39e12dfad459d1d521cd94a5aea205b3147ff20becd326e1def432dc75fc5f911548207504316d4f46d9a4d7@127.0.0.1:37233?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - ERROR[01-02|00:30:29.306] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.307] HTTP server started                      endpoint=127.0.0.1:6133 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.339] New local node record                    seq=1,735,777,829,311 id=1582b1f609c4727e ip=10.1.0.31 udp=42184 tcp=35351
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.341] New local node record                    seq=1,735,777,829,312 id=1582b1f609c4727e ip=10.1.0.31 udp=42184 tcp=35351
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - 2025-01-02T00:30:29.496Z	INFO	0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133	status-cli/util.go:62	existing account, key UID: 0x94a79a7074a5cbfc27a6ee4fbe303fb7340f49f6a4bd14ab7757940237918d9e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:29.502] New local node record                    seq=1,735,777,829,313 id=1582b1f609c4727e ip=10.1.0.31 udp=42184 tcp=35351
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - 2025-01-02T00:30:29.519Z	INFO	0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133	status-cli/util.go:93	messenger started, public key: 0x04beaa2e3907a845a32dbdfb857681344504297dffedcd46a33dec97fdc9f98e5f3201bf74a2630ed1d33e8ce33bff8fe842d38219664c0255e416264dc280cd28
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - INFO [01-02|00:30:30.334] New local node record                    seq=1,735,777,825,319 id=9f1d9b14b645ebbf ip=172.183.175.192 udp=37695 tcp=37765
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02F47B0AC659301B20823022CC4F230E3319EAD14F8E3EDBFEF4A9003F6FBFFB8D_6132 - 2025-01-02T00:30:30.534Z	INFO	0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135', '-kid', '0xc57c8da211d8ee6ddf284e8515416793a2f77952051c57e437c1331d8b967571']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - 2025-01-02T00:30:31.238Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - -name 0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - -key-uid 0xc57c8da211d8ee6ddf284e8515416793a2f77952051c57e437c1331d8b967571
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - 2025-01-02T00:30:31.238Z	INFO	0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.300] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.309] New local node record                    seq=1,727,788,954,796 id=7765e02f60d69100 ip=127.0.0.1 udp=0 tcp=40535
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.309] Started P2P networking                   self="enode://d42e32353d012032d56bf227c8b1ed5e78b14a4d3b6e6d7dc251f569ee38c418d232fa638a4b0edf0c21237698cb8535b44c75979076926193275ed8db99a5d8@127.0.0.1:40535?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - ERROR[01-02|00:30:31.310] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.310] HTTP server started                      endpoint=127.0.0.1:6135 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.344] New local node record                    seq=1,735,777,831,315 id=124fdeb71a609bdf ip=10.1.0.31 udp=43798 tcp=43021
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.349] New local node record                    seq=1,735,777,831,316 id=124fdeb71a609bdf ip=10.1.0.31 udp=43798 tcp=43021
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - 2025-01-02T00:30:31.503Z	INFO	0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135	status-cli/util.go:62	existing account, key UID: 0xc57c8da211d8ee6ddf284e8515416793a2f77952051c57e437c1331d8b967571
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:31.504] New local node record                    seq=1,735,777,831,317 id=124fdeb71a609bdf ip=10.1.0.31 udp=43798 tcp=43021
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - 2025-01-02T00:30:31.529Z	INFO	0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135	status-cli/util.go:93	messenger started, public key: 0x046b7f5a61f23c59ae579fadf755cacc96ff369ddb41497b61425ef724a286434eb084f272ead3a47061b7a77e2c8f00fbabb1265ce9732e70cb53565804238090
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:32.318] New local node record                    seq=1,735,777,817,303 id=c9958765038cf26c ip=172.183.175.192 udp=34568 tcp=37760
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - INFO [01-02|00:30:32.343] New local node record                    seq=1,735,777,827,325 id=ff13e6b5eba89c53 ip=172.183.175.192 udp=54136 tcp=37766
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02BF80242989F350415573D13A81866FF9E9F97BD335480CE4E4EFB74692CA032C_6137 - 2025-01-02T00:30:32.534Z	INFO	0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 4 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - INFO [01-02|00:30:34.335] New local node record                    seq=1,735,777,829,314 id=1582b1f609c4727e ip=172.183.175.192 udp=42184 tcp=37767
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036B8C7EA8E7A0651A39C433B661B77EB9FE31B8DB28DFDA593C30526AC726F5DE_6139 - INFO [01-02|00:30:34.351] New local node record                    seq=1,735,777,819,328 id=6862f44e235c7a3a ip=172.183.175.192 udp=35697 tcp=37763
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0252E55CAC8BE876AE9E102CF207CA6EF5BE2EFF8FEAAFB79BAF7B37311D530074_6133 - 2025-01-02T00:30:34.520Z	INFO	0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - INFO [01-02|00:30:36.335] New local node record                    seq=1,735,777,831,318 id=124fdeb71a609bdf ip=172.183.175.192 udp=43798 tcp=37768
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X0292E28506F89DEF6BEF386DE97D7344E7B4AFE5E8281E1CB2318CF83EC2C96CE5_6135 - 2025-01-02T00:30:36.529Z	INFO	0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.steps.common:common.py:39 Entering context manager: add_latency
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:40910 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035E0AF74F39FB6E72C21C1484305DBE5B028A72C3637C333C3C963F276944EE96_6138 - INFO [01-02|00:30:37.318] New local node record                    seq=1,735,777,817,304 id=c9958765038cf26c ip=172.183.175.192 udp=34568 tcp=37760
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:25 Received response: {"jsonrpc":"2.0","id":1,"result":{"id":"0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117","memberRole":0,"isControlNode":false,"verified":false,"joined":false,"joinedAt":0,"spectated":false,"requestedAccessAt":0,"name":"vac_qa_community_1","description":"vac_qa_community_1","introMessage":"","outroMessage":"","tags":[],"chats":{"71a07c50-403e-4a07-a026-2b159d6e95b4":{"id":"71a07c50-403e-4a07-a026-2b159d6e95b4","name":"general","color":"#ffffff","emoji":"","description":"General channel for the community","members":null,"permissions":{"access":1},"canPost":false,"canView":false,"canPostReactions":false,"viewersCanPostReactions":false,"position":0,"categoryID":"","tokenGated":false,"hideIfPermissionsNotMet":false,"missingEncryptionKey":false}},"categories":{},"images":null,"permissions":{"access":3},"members":{"0x04077490ccefaa54a51814da87ee4f0f303172ed429f23e433a2858e3e8be439cc2c912df5d6003359c1ed0ff9cccb078e847f8126310f4e66bb8c2b897756892d":{"last_update_clock":1735344983},"0x04f9be3ecf73726466655b09c261c7cc3b5bd99ca4b5b10fee6768970ee93e7f004ac362e30b66944c16a1137de5d6955d0ec3a80e72970eb0c3f032985ab08cc0":{"roles":[1]}},"canRequestAccess":true,"canManageUsers":false,"canDeleteMessageForEveryone":false,"canJoin":false,"color":"#ffffff","isMember":false,"muted":false,"muteTill":"0001-01-01T00:00:00Z","adminSettings":{"pinMessageAllMembersEnabled":false},"encrypted":false,"pendingAndBannedMembers":{},"tokenPermissions":{},"communityTokensMetadata":null,"activeMembersCount":0,"pubsubTopic":"","pubsubTopicKey":"","shard":null,"lastOpenedAt":0,"clock":1735344987503}}

[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:40910 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:40910 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:40910 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:40910 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:40910 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:40910 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.steps.common:common.py:44 Exiting context manager: add_latency
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testfetchcommunity_test_fetch_community_with_latency_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: first_node
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x03600e1152158616f5e0b6b8213bd9a2a0b52410f2aef58398f02c1b681fbad117_6131
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02afc3fe72d25d01723d97275041e5c000248ca36851c4b7947623561dc957f618_6136
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x035e0af74f39fb6e72c21c1484305dbe5b028a72c3637c333c3c963f276944ee96_6138
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x036b8c7ea8e7a0651a39c433b661b77eb9fe31b8db28dfda593c30526ac726f5de_6139
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0226cbc810a4c88c49da9920b74287463556974170eb628cc791639263b7c9e324_6130
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x022c9bb111279b1264eb3c86d11baf886354bcfed8e7b46db96871686f9f91d96e_6134
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02f47b0ac659301b20823022cc4f230e3319ead14f8e3edbfef4a9003f6fbffb8d_6132
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02bf80242989f350415573d13a81866ff9e9f97bd335480ce4e4efb74692ca032c_6137
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0252e55cac8be876ae9e102cf207ca6ef5be2eff8feaafb79baf7b37311d530074_6133
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x0292e28506f89def6bef386de97d7344e7b4afe5e8281e1cb2318cf83ec2c96ce5_6135